concrete/tools/permissions/access/entity/types/submitter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+defined('C5_EXECUTE') or die("Access Denied.");
+use Concrete\Core\Permission\Access\Entity\SubmitterEntity as SubmitterPermissionAccessEntity;
+
+if (\Loader::helper('validation/token')->validate('process')) {
```

Comment:
`Loader::helper('handle')` has been deprecated.
You should use `Core::make('helper/handle')`

Reply:
that´s right, but it should also be changed in all other access entity types!

Reply:
> that´s right, but it should also be changed in all other access entity types!

Sure!